intellij-plugin-structure/structure-intellij/src/main/java/com/jetbrains/plugin/structure/intellij/problems/LevelRemappingPluginCreationResultResolver.kt

```diff
@@ -0,0 +1,68 @@
+package com.jetbrains.plugin.structure.intellij.problems
+
+import com.jetbrains.plugin.structure.base.plugin.PluginCreationFail
+import com.jetbrains.plugin.structure.base.plugin.PluginCreationResult
+import com.jetbrains.plugin.structure.base.plugin.PluginCreationSuccess
+import com.jetbrains.plugin.structure.base.plugin.PluginProblem
+import com.jetbrains.plugin.structure.base.problems.ReclassifiedPluginProblem
+import com.jetbrains.plugin.structure.intellij.plugin.IdePlugin
+import kotlin.reflect.KClass
+
+class LevelRemappingPluginCreationResultResolver(private val delegatedResolver: PluginCreationResultResolver) : PluginCreationResultResolver {
+
+  private val remappedLevel: Map<KClass<*>, PluginProblem.Level> = mapOf(
+    IllegalPluginIdPrefix::class to PluginProblem.Level.WARNING,
+    InvalidSinceBuild::class to PluginProblem.Level.WARNING)
```

Comment:
Why is it needed to change the level of `InvalidSinceBuild`?

Reply:
Resolved in the class API and unit tests.